### PR TITLE
[MKT-315]: feat/star wars campaign

### PR DESCRIPTION
**Pull request by xabg2**
Add Star Wars campaign. This campaign includes:
* HeroSection in home
* Top banner
* Bottom banner
* Home banner for mobile view
* Pricing banner

### CODE REVIEW

File: src/components/lifetime/PriceBannerForCampaigns.tsx [hunk 1]
```diff
@@ -9,18 +10,28 @@ export const PriceBannerForCampaigns = ({ textContent }) => {
   const router = useRouter();
 
   function handleOnClick() {
-    router.push('/lifetime');
+    window.location.hash = 'billingButtons';
```
Comment (aboredvaro) [inline src/components/lifetime/PriceBannerForCampaigns.tsx]: It is preferred to use `href="#billingButtons"` in the HTML element than using JavaScript window.location.hash to scroll to that element.

File: src/components/virus-scanner/HeroSection.tsx [hunk 2]
```diff
@@ -41,7 +39,7 @@ const HeroSection = ({ textContent }) => {
       body: formdata,
     };
 
-    fetch(`${FILE_SCANNER_URL}/filescan`, requestOptions)
+    fetch(`https://clamav.internxt.com/filescan`, requestOptions)
```
Comment (aboredvaro) [inline src/components/virus-scanner/HeroSection.tsx]: If we make this public, we should restrict to requests only made from "internxt.com" origin, so other people are not able to use this as free API by inverse engineering it.

File: src/components/auth/SignUpInline.tsx [hunk 3]
```diff
@@ -117,7 +118,7 @@ export default function SignUpInline(props: SignUpInlineProps) {
           />
         </div>
 
-        <span className="w-full text-sm text-gray-50 sm:text-left">
+        <span className={`w-full text-sm ${props.textColor ?? 'text-gray-50'} sm:text-left`}>
```
Comment (aboredvaro) [inline src/components/auth/SignUpInline.tsx]: This is [not recommended by TailwindCSS](https://tailwindcss.com/docs/content-configuration#dynamic-class-names), colors can be conditional, but if passing colors as prop they should be defined (as string, string array or string object), so dev cannot enter a random color.

**Review by aboredvaro — CHANGES_REQUESTED** [review]

File: src/components/virus-scanner/HeroSection.tsx (hunk 2 above)
Comment (xabg2) [reply src/components/virus-scanner/HeroSection.tsx]: You're right, it slipped through. I must change the name of the variable...

Nice catch!

**Review by aboredvaro — APPROVED** [review]

**Pull request merged**